develop/sounds/using-sounds.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Playing SoundEvents
+description: Learn how to play sound events.
+---
+
+# Playing SoundEvents
+
+Minecraft has a big selection of sounds which you can choose from. Check out the `SoundEvents` class to view all the vanilla sound event instances that Mojang has provided.
+
+## Using sounds in your mod
+
+Make sure to execute the `playSound()` method on the logical server side when using sounds!
```

Comment:
Maybe mention something about using the client's soundmanager to play sounds only on the client? PositionedSoundInstance etc.

Reply:
I was planning on doing an extra page for using SoundInstances and how to loop, transition and handle e.g. distances on those. Would be great if a networking page would exist for that tho 🤔